js/src/dom/selector-engine.js

```diff
@@ -7,9 +7,30 @@
 
 import { isDisabled, isVisible } from '../util/index.js'
 
-/**
- * Constants
- */
+const getSelector = element => {
```

Comment:
For consistency with `manipulator.js` and `event-handler.js` don't we need to use `function getSelector(element) {` here instead?

Reply:
As the index.js is the exact opposite, I am not so sure, which version should we keep

Reply:
Yeah OK I understand. So you can merge it as is I suppose.